### ISSUE TITLE
Request: Redshift

### ISSUE THREAD

**Issue by orschiro** [SomeElementary/PPA-Issues#2]
Dear @JustSomeDood,

Thank you so much for this repository! Mind adding the great [Redshift](http://www.theguardian.com/sustainable-business/2015/mar/20/key-to-happiness-connection-not-competition) tool to your list?

It allows to change the 'redness' of your display depending on your location and time. This makes the display much more readable. 

Thank you!

Robert

**Comment by JustSomeDood** [comment]
right, that's not really elementary specific which is what my ppa is for, but I can possibly package it for my general apps ppa

**Comment by orschiro** [comment]
Thanks very much!

Can you please point me to your general PPA?

Lazy type mode on sent from mobile

**Comment by JustSomeDood** [comment]
here you go https://launchpad.net/~justsomedood/+archive/ubuntu/justsomeapps

**Comment by orschiro** [comment]
> right, that's not really elementary specific which is what my ppa is for, but I can possibly package it for my general apps ppa btw the link you posted has nothing to do with the tool.

My fault. I updated my link.

> here you go https://launchpad.net/~justsomedood/+archive/ubuntu/justsomeapps

Thanks! I guess that [Gcolor3](https://github.com/Unia/gcolor3) would be also a suggestion for the general ppa than, right?

**Comment by JustSomeDood** [comment]
Anything that is not elementary specific, you can however still open issues here.  In fact I may just make this a general ppa issue tracker, since I have a few.  I'll just update the readme and ask that people mark their issues with [elementary] or [general] or whatever.  I have an interview tomorrow, i'll get it sorted after that.

**Comment by orschiro** [comment]
> I have an interview tomorrow, i'll get it sorted after that.

Success with that and thanks for your great work!
